_totp.md

```diff
@@ -0,0 +1,131 @@
+  "verified": true,
+  "verifiedAt": "2019-01-11T14:20:04.055Z"
+}]
+```
+
+Retrieves a list of authentication methods for the current user.
+
+### Request
+`GET https://api.uphold.com/v0/me/authentication_methods`
+
+### Response
+Returns an array of the current user's authentication methods.
+
+## Add authentication method
+
+```bash
+curl https://api.uphold.com/v0/me/authentication_methods/totp \
+  -X POST \
+  -H "Authorization: Bearer <token>" \
+  -H "Content-Type: application/json" \
+  -H 'OTP-Method-Id: <OTP-Method-Id>'
+  -H 'OTP-Token: <OTP-Token>'
+```
+
+> The above command returns the following JSON:
+
+```json
+{
+  "default": false,
+  "id": "3f8f8264-2f5e-4b2b-8333-473715ab039a",
+  "label": "Authenticator TOTP",
+  "type": "totp",
+  "url": "otpauth://totp/Uphold:han.solo@rebelalliance.org?algorithm=SHA1&digits=6&issuer=Uphold&period=30&secret=QRV62S3O6LXDB7FRKR4LMF3VGR6MZT7S",
+  "verified": false,
+  "verifiedAt": null
+}
+```
+
+### Request
+
+`POST https://api.uphold.com/v0/me/authentication_methods/totp`
+
+<aside class="notice">Requires the <code>OTP-Method-Id</code> header to be sent with the id of the phone associated with the user's profile information.</aside>
+<aside class="notice">Requires the <code>OTP-Token</code> header to be sent with a valid TOTP token, in cases where there was an already existing authentication method set in place.</aside>
+
+
+### Response
+
+Returns an object representing the authentication method created.
+
+## Verify authentication method
+
+```bash
+curl https://api.uphold.com/v0/me/authentication_methods/3f8f8264-2f5e-4b2b-8333-473715ab039a/verify \
+  -X POST \
+  -H "Authorization: Bearer <token>" \
+  -H "Content-Type: application/json" \
+  -d '{ "token": "<OTP-Token>" }'
+```
+
+> The above command returns the following JSON:
+
+```json
+{
+  "default": false,
+  "id": "3f8f8264-2f5e-4b2b-8333-473715ab039a",
+  "label": "Authenticator TOTP",
+  "type": "totp",
+  "verified": true,
+  "verifiedAt": "2019-02-11T14:31:48.485Z"
+}
+```
+
+### Request
+
+`POST https://api.uphold.com/v0/me/authentication_methods/3f8f8264-2f5e-4b2b-8333-473715ab039a/verify`
+
+
+### Response
+
+Returns an object representing the verified authentication method.
+
+## Remove authentication method
+
+```bash
+curl https://api.uphold.com/v0/me/authentication_methods/3f8f8264-2f5e-4b2b-8333-473715ab039a \
+  -X DELETE \
+  -H "Authorization: Bearer <token>" \
+  -H "Content-Type: application/json" \
+  -H 'OTP-Token: <OTP-Token>'
+```
+
+> The above command does not return a JSON response.
+
+### Request
+
+`DELETE https://api.uphold.com/v0/me/authentication_methods/3f8f8264-2f5e-4b2b-8333-473715ab039a`
+
+<aside class="notice">Requires the <code>OTP-Token</code> header to be sent with a valid TOTP token for an existing authentication method. This means that you cannot delete the last verified method of a user.</aside>
+
+### Response
+
+Returns an http status code of 204 in case of success and no JSON body.
```

Comment:
Perhaps wrap "204" in a `<code>` tag? Also, I'd add a comma before the "and no JSON body", just to make the sentence structure more evident.

Reply:
Please also capitalize "HTTP".